DataFormats/FTLRecHit/interface/FTLRecHit.h

```diff
@@ -0,0 +1,109 @@
+#ifndef DATAFORMATS_FTLRECHIT_H
+#define DATAFORMATS_FTLRECHIT_H 1
+
+#include "DataFormats/CaloRecHit/interface/CaloRecHit.h"
+#include <vector>
+
+/** \class FTLRecHit
+ *  
+ * based on EcalRecHit
+ *
+ * \author Lindsey Gray
+ */
+
+class FTLRecHit : public CaloRecHit {
+public:
+  typedef DetId key_type;
+
+  // FTLEE recHit flags
+    enum Flags { 
+          kGood=0,                   // channel ok, the energy and time measurement are reliable
+          kPoorReco,                 // the energy is available from the UncalibRecHit, but approximate (bad shape, large chi2)
+          kOutOfTime,                // the energy is available from the UncalibRecHit (sync reco), but the event is out of time
+          kFaultyHardware,           // The energy is available from the UncalibRecHit, channel is faulty at some hardware level (e.g. noisy)
+          kNoisy,                    // the channel is very noisy
+          kPoorCalib,                // the energy is available from the UncalibRecHit, but the calibration of the channel is poor
+          kSaturated,                // saturated channel (recovery not tried)
+          kDead,                     // channel is dead and any recovery fails
+          kKilled,                   // MC only flag: the channel is killed in the real detector
+          kWeird,                    // the signal is believed to originate from an anomalous deposit (spike) 
+          kDiWeird,                  // the signal is anomalous, and neighbors another anomalous signal  
+                                     //
+          kUnknown                   // to ease the interface with functions returning flags. 
```

Comment:
these look like a copy-paste of ecal.
If you do not have any use for this enum, please remove